tests/publiccloud/slem_basic.pm

```diff
@@ -56,20 +62,19 @@ sub run {
 
     select_serial_terminal();
 
-    if ($qam) {
+    if ($qam or $args->{my_instance}) {
         $instance = $self->{my_instance} = $args->{my_instance};
         $provider = $self->{provider} = $args->{my_provider};
     } else {
         $provider = $self->provider_factory();
         $instance = $self->{my_instance} = $provider->create_instance(check_guestregister => 0);
     }
-    $provider->{username} = 'suse';
 
     # On SLEM 5.2+ check that we don't have any SELinux denials. This needs to happen before anything else is ongoing
     $self->check_avc() unless (is_sle_micro('=5.1'));
 
     my $test_package = get_var('TEST_PACKAGE', 'jq');
-    registercloudguest($instance);
+    registercloudguest($instance) unless (is_scc_registered);
```

Comment:
This condition is needed due to previous `registration` module work.

Reply:
Removed in PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19186